### DISCUSSION TITLE
欢迎来到 哔哩 论坛👋👋

### DISCUSSION THREAD

**Discussion by Richasy** [Richasy/Bili.Uwp#48] [通知]
## 你可以在论坛进行以下活动...

- 问关于应用或 BiliBili 的问题 💭
- 讨论 💬
- 变成好奇宝宝 💡
- 反馈问题 📑
- 欢迎新成员 🙋
- 分享关于哔哩的一切 😆

所以，哔哩哔哩，干杯 ( ゜- ゜)つロ

---

*但请千万不要发布任何政治言论，不要涉及黄赌毒，网络不是法外之地👮*

**Comment by Groupguanfang** [comment]
先占个楼
b站官方电视版太难用了 大佬试试开发个安卓app试试？
嘿嘿嘿

**Reply by Richasy** [reply]
呀……不会安卓开发😂

**Reply by minortex** [reply]
去找哔哩哔哩tv版，有个非官的挺好用

**Comment by ghost** [comment]
我怎么感觉 我播放视频有点卡顿 是哪里出了问题么

**Comment by tm0210** [comment]
(つ^~^)/○吃月饼(=ó∀ò)o中秋快乐

**Comment by ZovD2002** [comment]
有时候视频点进去是黑的，不显示播放，有时候是开全屏之后黑的

**Comment by SamllStrong** [comment]
我想问问这个UWP怎么看港澳台的番剧？如果能直接看就方便多了，和手机修改版一样，可以直接点开观看

**Comment by yyx-dev** [comment]
请问这个就是微软商店里的哔哩吗？（不可下载的第三方的哔哩）几经周折找到这个，才特意到GitHub上看看果然有。感谢UP

**Reply by yyx-dev** [reply]
果然是的，UP在微软商店的网站链接被我找到了，和GitHub的ID一样，谢谢喽希望我以后也能成为和UP一样的大佬

**Comment by michael-eddy** [comment]
老哥，我手头已经有个可以实现跨平台的客户端的，你看有没有机会一起合作？[这是预览版链接](https://michael-eddy.github.io/#avalonia-bilibili)

**Reply by Richasy** [reply]
嗨，很高兴看到你使用新的跨平台框架进行开发。我们可以互相借鉴，不知道你预期的合作方式是？

**Reply by michael-eddy** [reply]
😂目前可能是以功能验证为主，毕竟有些接口i可能已经是比较旧的，而且有些功能还是有缺失的地方。但可能当前前端效果目前还有点糟糕，所以当前我的重心是以前端优化为主。如果可以我们先加个扣交流交流？

**Reply by michael-eddy** [reply]
> 嗨，很高兴看到你使用新的跨平台框架进行开发。我们可以互相借鉴，不知道你预期的合作方式是？

而且除了极个别对于跨平台比较苛刻的功能比如弹幕、浏览器嵌入，有C++的同志帮忙通过外挂实现效果，所以本人对于这个框架还是比较看好的

**Reply by Richasy** [reply]
好的，你可以把qq发到我的邮箱：[thansy@foxmail.com](mailto:thansy@foxmail.com)，期待和你的交流

**Comment by FlyongX** [comment]
不是科班出身的微软大佬🤩太强了，加油！

**Comment by Chenxl007** [comment]
2.2207.1.0版本，播放页全窗口模式，左上角没有给个返回按钮，都得用键盘操作么？鼠标的点按操作无法设置，单击或者双击执行什么操作？现在就默认单击暂停/播放，双击切换播放器显示模式？操作上有点麻烦，记得之前版本是有返回按钮及鼠标操作设置的！？

**Comment by tianyehuakai** [comment]
最新的10月版本怎么评论不了

**Comment by clownfish-1** [comment]
有没有qq群呀，更新之后能在qq群里下载的那种

**Comment by tianyehuakai** [comment]
这是来自QQ邮箱的假期自动回复邮件。
 
您好，我最近正在休假中，无法亲自回复您的邮件。我将在假期结束后，尽快给您回复。

**Comment by YHLShaw** [comment]
这个项目clone下来后 怎么编译不过，APP.csproject 要选编译的模式x86 还是AnyCPU

**Reply by Richasy** [reply]
选x64
